### PR TITLE
#13128: Fixed incorrect position of {pos} string in ColumnSchemaBuilder __toString

### PR DESCRIPTION
**Pull request by usecomposable**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | unknown
| Fixed issues  | #13128

### CODE REVIEW

**Comment by df2** [comment]
Can this please be merged into master?

**Comment by dynasource** [comment]
if the append string is not inserted at last, its not really an append string anymore right ;)

**Comment by cebe** [comment]
Yeah but still it makes sense to change it because DBMS syntax is this way. It may be useful to point that out in the docs though.

**Comment by dynasource** [comment]
does this "append" typology match any mysql typology or is it chosen by us. Tried to find a clarifying reference.

**Comment by df2** [comment]
Basically `append` is used to append to the column definition. The position indicator should always be last in the SQL statement.

MySql: http://dev.mysql.com/doc/refman/5.7/en/alter-table.html

```
| CHANGE [COLUMN] old_col_name new_col_name column_definition
      [FIRST|AFTER col_name]
```

Cubrid: http://www.cubrid.org/manual/91/en/sql/schema/table.html#change-modify-clauses

```
    CHANGE [COLUMN | CLASS ATTRIBUTE ] old_col_name new_col_name column_definition
             [FIRST | AFTER col_name]
  | MODIFY [COLUMN | CLASS ATTRIBUTE] col_name column_definition
             [FIRST | AFTER col_name]
```

Oracle: It seems it was a mistake to allow the position option. I have removed it from the oci `__toString`.

**Comment by df2** [comment]
Not sure why these tests are failing... help much appreciated.

**Comment by samdark** [comment]
Tests are not failing, it's codeclimate, not travis.

File: tests/framework/db/QueryBuilderTest.php [hunk 1]
```diff
@@ -928,6 +928,61 @@ public function columnTypes()
                     'cubrid' => "int NOT NULL AUTO_INCREMENT PRIMARY KEY COMMENT 'test comment'",
                 ],
             ],
+            [
+                Schema::TYPE_PK . " FIRST",
```
Comment (samdark) [inline tests/framework/db/QueryBuilderTest.php]: Should it be special for Oracle which doesn't support it?

**Review by samdark — CHANGES_REQUESTED** [review]

File: tests/framework/db/QueryBuilderTest.php (hunk 1 above)
Comment (df2) [inline tests/framework/db/QueryBuilderTest.php]: Hmm, I think then with Oracle, adding a position modifier will have no effect, and thus the tests are correct. It is the same as Postgres in that respect.

Comment (samdark) [inline tests/framework/db/QueryBuilderTest.php]: OK.

**Review by samdark — APPROVED** [review]

**Review by SilverFire — APPROVED** [review]

**Review by cebe — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**